README.md

```diff
@@ -1,24 +1,180 @@
-# README
+<!-- TABLE OF CONTENTS -->
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+<details>
+  <summary>📗 Table of Contents</summary>
+  <ul>
+    <li><a href="#-car-rental-front-end-">📖 Book-an-Appointment </a></li>
+    <li><a href="#-built-with-">🛠 Built With</a></li>
+    <li><a href="#-getting-started-">💻 Getting Started</a></li>
+    <li><a href="#-authors-">👥 Authors </a></li>
+    <li><a href="#-future-features-">🔭 Future Features</a></li>
+    <li><a href="#-contributing-">🤝 Contributing</a></li>
+    <li><a href="#️-show-your-support-">⭐️ Show your support </a></li>
+    <li><a href="#-acknowledgments-">🙏 Acknowledgments </a></li>
+    <li><a href="#-license-">📝 License</a></li>
+  </ul>
+</details>
 
-* Ruby version
+<br>
 
-* System dependencies
+<!-- PROJECT DESCRIPTION -->
 
-* Configuration
+# 📖 Book-an-Appointment <a name="about-project"></a>
 
-* Database creation
+**Book-an-Appointment** is an application for cars. Create an account, and find and book your dream car today.
 
-* Database initialization
 
-* How to run the test suite
+## 🛠 Built With <a name="built-with"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+### Tech Stack <a name="tech-stack"></a>
 
-* Deployment instructions
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://www.rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
 
-* ...
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Handles logic for storing and retrieving data from the database
+- Handles logic for returning data to the frontend
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🪵 Documentation <a name="key-features"></a>
+
+- [Api docs]()
```

Comment:
When live demo is available don't forget to link to the docs page (the swagger index.html)...

[OPTIONAL] - Also consider adding info about accessing docs in local deployment